examples/karpenter-fargate/README.md

```diff
@@ -0,0 +1,210 @@
+# EKS Cluster with Karpenter Cluster Autoscaler running on Fargate
```

Comment:
We should start with https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/examples/fargate-serverless/main.tf as the baseline for this example and update to show the relevant aspects that we are trying to highlight that are different

Reply:
"Cluster Autoscaler" is not something the official docs mention.
plus is not about the cluster, but about the worker nodes

Reply:
I reused what's in https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/examples/karpenter/README.md?plain=1#L1.

I will update the `README.md` file following @bryantbiggs recommendation.